packages/cli/src/index.ts

```diff
@@ -447,7 +460,12 @@ const main = async () => {
     }
 
     if (user.id === scope || user.email === scope || user.username === scope) {
-      delete client.config.currentTeam;
+      if (user.version === 'northstar') {
+        output.error('You cannot set your Personal Account as the scope.');
+        return 1;
+      } else {
+        delete client.config.currentTeam;
+      }
```

Comment:
**suggestion (non-blocking):** nitpick - The `else` isn't necessary, but I'm fine if you wanna leave it.‏

```suggestion
      }

      delete client.config.currentTeam;
```